test/extended/util/cvo.go

```diff
@@ -0,0 +1,57 @@
+package util
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configclientv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+)
+
+// WaitForClusterProgression waits for a cluster-level configuration change to propagate across all operators.
+// This polls all ClusterVersion objects and waits until the following states are stable:
```

Comment:
what does ClusterVersion have to do w/ the MCO rolling out a change to node config?

Reply:
Fixed to be `ClusterOperator` - aim is to use those to verify the cluster has reached a stable state.

Reply:
godoc still says ClusterVersion